src/audio/volume/volume.c

```diff
@@ -352,10 +352,6 @@ void volume_prepare_ramp(struct comp_dev *dev, struct vol_data *cd)
 	 * four presets are used.
 	 */
 	if (cd->initial_ramp < VOL_RAMP_UPDATE_THRESHOLD_FASTEST_MS)
-		ramp_update_us = VOL_RAMP_UPDATE_FASTEST_US;
```

Comment:
@singalsu would it be better here making ```ramp_update_us``` a Kconfig build time constant. he Kconfig help should explain the benefits and trade off for selecting the value.

Reply:
There could be choice for smallest update step 125 us / 250 us / 500 us / 1000 us, and the thresholds for selecting the larger steps could be replaced with an equation that is roughly like current if - else code. But the problem is that in my opinion anything but the fastest updated ramp sound quite annoying. So I'd not change the default 125 us.

Should I make such PR?